apps/admin-gui/src/app/admin/components/attr-def-list/attr-def-list.component.ts

```diff
@@ -82,4 +95,20 @@ export class AttrDefListComponent implements OnChanges, AfterViewInit {
     }
     return `${this.selection.isSelected(row) ? 'deselect' : 'select'} row ${row.id + 1}`;
   }
+
+  onRowClick(attDef: AttributeDefinition) {
+    const dialogRef = this.dialog.open(EditAttributeDefinitionDialogComponent, {
+      width: '500px',
```

Comment:
I think this can be even a bit wider, maybe 700px?

Reply:
Done